apix-impl/src/integration-test/java/eu/xenit/apix/tests/BaseTest.java

```diff
@@ -87,6 +90,20 @@ protected boolean removeTestNode(NodeRef nodeRef) {
         return success;
     }
 
+    protected File createTextFileWithContent(String fileName, String content) throws IOException {
+        if (!fileName.endsWith(".txt")) {
+            fileName += ".txt";
+        }
+        File textFile = new File(fileName);
+        textFile.deleteOnExit();
+        textFile.createNewFile();
+        PrintWriter writer = new PrintWriter(fileName, "UTF-8");
+        String contentString = content;
```

Comment:
You have both `contentString` and `content` with the same value. Why not delete `contentString`?